src/DependencyInjection/Configuration.php

```diff
@@ -408,10 +410,12 @@ public function getConfigTreeBuilder()
                         ->end()
                         ->arrayNode('extra_javascripts')
                             ->defaultValue([])
+                            ->info('javascripts to add to the defaults')
                             ->prototype('scalar')->end()
                         ->end()
                         ->arrayNode('remove_javascripts')
                             ->defaultValue([])
+                            ->info('javascripts to remove from the defaults')
```

Comment:
Could you rename it like this: "Named javascript files get removed from the ???" I don't know, but the don't get removed from the `defaults`, the get removed from the result of all JS files..


hmm, @greg0ire can you help? Or shall we keep it like this?

Reply:
I changed this so it says "javascripts to add to the page". More clear I think because like you said this is not changing the defaults directly, but the full list of javascripts that will be added.